src/main/java/org/sonarsource/scanner/maven/bootstrap/ScannerBootstrapper.java

```diff
@@ -57,49 +57,49 @@ public RunnerBootstrapper(Log log, MavenSession session, EmbeddedRunner runner,
   public void execute() throws IOException, MojoExecutionException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.IOException, org.apache.maven.plugin.MojoExecutionException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1160)